lib/omnibus-software/version.rb

```diff
@@ -0,0 +1,3 @@
+module OmnibusSoftware
+  VERSION = "5.0.0"
```

Comment:
nitpick: the upstream version is `4.0.0`: https://github.com/chef/omnibus-software/blob/c47c8200a9d112db2d73b4d7252bcf9dfd164ca6/lib/omnibus-software/version.rb#L2

we should probably use that one? (or maybe keep the same old `3.0.0` version)